src/core/src/op/convert.cpp

```diff
@@ -212,6 +229,10 @@ bool Convert::has_evaluate() const {
         return et == element::f16 || et == element::bf16 || et == element::f32 || et == element::f4e2m1;
     };
 
+    const auto can_convert_f8e8m0 = [](const element::Type& et) {
+        return et == element::f16 || et == element::bf16 || et == element::f32 || et == element::f8e8m0;
```

Comment:
```suggestion
        et == element::f16 || et == element::bf16 || et == element::f32;
```
Can we make dedicated check for these types and use it in these lambdas?

Reply:
Done